.github/workflows/release.yaml

```diff
@@ -0,0 +1,25 @@
+name: Push Tag
```

Comment:
Tag sounds like a image tag to me.
How about changing it to `release library`? Since the file name is `realse.yaml`